eng/Versions.props

```diff
@@ -72,6 +72,7 @@
     <!-- for templates -->
     <MicrosoftExtensionsHttpResiliencePackageVersionForNet8>$(MicrosoftExtensionsHttpResiliencePackageVersion)</MicrosoftExtensionsHttpResiliencePackageVersionForNet8>
     <MicrosoftExtensionsHttpResiliencePackageVersionForNet9>$(MicrosoftExtensionsHttpResiliencePackageVersion)</MicrosoftExtensionsHttpResiliencePackageVersionForNet9>
+    <MicrosoftAspNetCorePackageVersionForNet9>9.0.0-rc.2.24474.3</MicrosoftAspNetCorePackageVersionForNet9>
```

Comment:
In order for this version to flow or get updated automatically, We should probably use an actual package here that ships from aspnetcore so that each time we update package versions it will get associated to a real package and get bumped. Can't we reuse any package that we already depend on?

Reply:
I don't see any others in this file.

Reply:
Chatted offline, this is the only ASP.NET Core 9 ref we have and we'll revisit automating updating of this later on.